eng/native/init-distro-rid.sh

```diff
@@ -41,7 +41,7 @@ initNonPortableDistroRid()
             # We have forced __PortableBuild=0. This is because -portablebuld
             # has been passed as false.
             if (( isPortable == 0 )); then
-                if [[ "${ID}" == "rhel" || "${ID}" == "alpine" ]]; then
+                if [[ "${ID}" == "rhel" || "${ID}" == "rocky" || "${ID}" == "alpine" ]]; then
```

Comment:
Seems both single or double square brackets are allowed to evaluate expressions. I mention it because the main version only had single square brackets.

Source: http://mywiki.wooledge.org/BashFAQ/031

Reply:
To be honest, I didn't even notice that single brackets where used on main, I only wished to add `rocky` as a valid alternative to the existing test. It doesn't have any functional impact here, as double brackets only give access to the extended Bash test syntax.

Reply:
It seems double brackets are more correct. https://stackoverflow.com/a/2188369/8816314

If we have to do any corrections, we would have to add them to main, not here. But I think single brackets work too, if I'm understanding those two sources correctly.

Reply:
Single brackets are compatible with /bin/sh (e.g. standard POSIX tests), whereas double brackets are a bash extension. I assume it has since been decided to not use bash extensions on main, but I would leave it as it is on servicing.